### ISSUE TITLE
Configuring limit_memory_soft and limit_memory_hard

### ISSUE THREAD

**Issue by etechsam** [odoo/odoo#107799]
Impacted versions: 14

Steps to reproduce:
Assuming that I have 4vCPU and 8Gb RAM
Depending on the official documentation, the rule of thumbs is: number of worker = (#CPU * 2) + 1 = 9
the first thing I don't understand is that: this "+ 1" is it for the cron or for the liveChat
if it is for cron then the max_cron_threads is for what?

secondly, I'm confused on how we should configure limit_memory_soft and limit_memory_hard
if we run odoo-bin --help, it says that the these limits are per worker. 
In other words, the value we set for these parameters is for 1 worker.

But I see in some forums, that actually we have to consider the number of the worker to calculate these limits.
So instead of having just
limit_memory_soft = 2048 * 1024 * 1024
limit_memory_hard = 2560 * 1024 * 1024
We have set them like:
limit_memory_soft = 2048 * workers * 1024 * 1024
limit_memory_hard = 2560 * workers * 1024 * 1024

So what is the right way to configure these limits?
Is there some tools that can help to configure them?

Thank you

**Comment by lavrenkov-sketch** [comment]
I also need help on this issue

**Comment by dungnm8** [comment]
I same question. Please help me

**Comment by TGha** [comment]
indeed, the documentation and some forums are not consistent. Need clarification
